### PR TITLE
[WIP] Content turn gear to models

### PR DESCRIPTION
**Pull request by TheHollidayInn**
[//]: # (Note: See http://habitica.wikia.com/wiki/Using_Habitica_Git#Pull_Request for more info)

[//]: # (Put Issue # or URL here, if applicable. This will automatically close the issue if your PR is merged in)
Fixes put_issue_url_here

### Changes
[//]: # (Describe the changes that were made in detail here. Include pictures if necessary)

[//]: # (Put User ID in here - found in Settings -> API)

----
UUID:

### CODE REVIEW

File: website/client-old/js/controllers/tasksCtrl.js [hunk 1]
```diff
@@ -215,8 +215,12 @@ habitrpg.controller("TasksCtrl", ['$scope', '$rootScope', '$location', 'User','N
      ------------------------
      */
 
-    $scope.$watch('user.items.gear.owned', function(){
-      $scope.itemStore = Shared.updateStore(User.user);
+    $scope.$watch('user.items.gear.owned', function() {
+      $scope.itemStore = Shared.updateStore(User.user, Content);
```
Comment (paglias) [inline website/client-old/js/controllers/tasksCtrl.js:219]: why is the store updated twice?

File: website/client-old/js/services/sharedServices.js [hunk 2]
```diff
@@ -8,6 +8,36 @@ angular.module('habitrpg')
...
+      url: '/api/v3/content',
+    })
+    .then(function (response) {
+      Shared.content = response.data.data;
+      for (var key in Shared.content.gear.flat) {
+        var gearItem = Shared.content.gear.flat[key];
+        var text = gearItem.text;
+
+        gearItem.textlocaleKey = gearItem.text;
+        gearItem.text = function (language) {
```
Comment (paglias) [inline website/client-old/js/services/sharedServices.js:25]: shouldn't already be translated from the server?

File: website/common/script/content/gear/sets/armoire.js [hunk 3]
```diff
@@ -5,6 +5,8 @@ let armor = {
   lunarArmor: {
     text: t('armorArmoireLunarArmorText'),
     notes: t('armorArmoireLunarArmorNotes', { str: 7, int: 7 }),
+    textlocaleKey: 'armorArmoireLunarArmorText',
```
Comment (paglias) [inline website/common/script/content/gear/sets/armoire.js:8]: I don't think this is necessary, the server should return already translated texts

File: website/server/controllers/api-v3/content.js [hunk 4]
```diff
@@ -112,13 +113,19 @@ api.getContent = {
     let content;
 
     // is the content response for this language cached?
-    if (cachedContentResponses[language] === true) {
-      content = await fs.readFile(`${CONTENT_CACHE_PATH}${language}.json`, 'utf8');
-    } else { // generate the response
+    // if (cachedContentResponses[language] === true) {
```
Comment (paglias) [inline website/server/controllers/api-v3/content.js:116]: I'm quite sure we want to keep caching

**Review by paglias — CHANGES_REQUESTED** [review]

**Comment by khipkin** [comment]
Suggestion: Replace 133 instances of "notesLocalKey" with "Notes**Locale**Key".

**Comment by TheHollidayInn** [comment]
Closing this because from the redesign we are able to import parts of the common files using Webpack. I think the new importing method will allow us to migrate to "CMS" like features on portions of the content at a time rather than the whole content collection.

**Pull request closed without merging**